main.go

```diff
@@ -45,7 +45,9 @@ func main() {
 	}
 	cs := kubernetes.NewForConfigOrDie(config)
 
-	watch, err := cs.CoreV1().Pods("").Watch(context.Background(), metav1.ListOptions{})
+	watch, err := cs.CoreV1().Pods("").Watch(context.Background(), metav1.ListOptions{
+		LabelSelector: "crashloopbackon = true",
```

Comment:
I thought the spaces around the equals weren't allowed, but I tested it and it works.